server/task_executor.go

```diff
@@ -84,7 +84,7 @@ func preMigration(ctx context.Context, server *Server, task *api.Task, migration
 		}
 	} else {
 		mi.Source = db.VCS
-		mi.Creator = vcsPushEvent.FileCommit.AuthorName
+		mi.Creator = vcsPushEvent.AuthorName
```

Comment:
@dragonly - wouldn't this assume all commits are coming from the same author? (which is not always the case)